### PR TITLE
Add remaining Travis CI stages to GitHub action workflow

### PR DESCRIPTION
**Pull request by hickeyma**
Add:
- static check
- build
- e2e tests

Partial #613

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          sudo apt-get update &&\
+          sudo apt-get install -y libseccomp-dev
+        
+          VERSION=v0.8.2 &&\
+          sudo mkdir -p /opt/cni/bin &&\
+          sudo wget -qO- https://github.com/containernetworking/plugins/releases/download/$VERSION/cni-plugins-linux-amd64-$VERSION.tgz \
+              | sudo tar xfz - -C /opt/cni/bin &&\
+          ls -lah /opt/cni/bin
+
+          VERSION=v1.0.0-rc8 &&\
```
Comment (saschagrunert) [inline .github/workflows/e2e.yml:42]: Can we use the latest release?

Comment (hickeyma) [reply .github/workflows/e2e.yml]: Sure, updated cni plugin version as well.

**Comment by hickeyma** [comment]
Thanks for review and merge @saschagrunert

**Pull request merged**